packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -353,8 +353,14 @@ class TextSpan extends DiagnosticableTree {
 
   @override
   List<DiagnosticsNode> debugDescribeChildren() {
```

Comment:
add a test case with a null child

Reply:
There's already a test for that.

Reply:
(It's the test that failed on Travis... Fixing.)